lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.tsx

```diff
@@ -102,21 +105,25 @@ export default function BasicLTILaunchApp() {
   /**
    * Helper to handle thrown errors from from API requests.
    *
-   * @param {Error} error - Error object from request.
-   * @param {ErrorState} state
-   * @param {boolean} [retry=true] - Can the request be retried?
-   * @param {string} [authURL] -
-   *   Authorization URL association with the API request
+   * @param error - Error object from request
+   * @param state
+   * @param [retry=true] - Can the request be retried?
+   * @param [authURL] - Authorization URL association with the API request
    */
-  const handleError = (error, state, retry = true, authURL) => {
+  const handleError = (
+    error: Error,
+    state: ErrorState,
+    retry = true,
+    authURL?: string
+  ) => {
     // Here we always set the authorization URL, but we could improve UX by
     // not setting it if the problem is not related to authorization (eg.
     // a network fetch error).
     setAuthURL(authURL || null);
 
     if (isLTILaunchServerError(error)) {
       setError(error);
-      setErrorState(/** @type {ErrorState} */ (error.errorCode));
+      setErrorState(error.errorCode as ErrorState);
```

Comment:
Is this just a non-null cast, if so you can use `!`.

Reply:
It's not :)